systemd/ceph-mds@.service

```diff
@@ -11,14 +11,15 @@ EnvironmentFile=-/etc/sysconfig/ceph
 Environment=CLUSTER=ceph
 ExecStart=/usr/bin/ceph-mds -f --cluster ${CLUSTER} --id %i --setuser ceph --setgroup ceph
 ExecReload=/bin/kill -HUP $MAINPID
-PrivateDevices=yes
+PrivateDevices=no
```

Comment:
@batrick thanks for your response
yes it is feasible, i just don't like the idea of adding new file.
ideally i would use ms_type param in /etc/ceph/ceph.conf to know posix/rdma and then set systemd accordingly.